md/api-extensions.md

```diff
@@ -23,6 +23,10 @@ REST API extensions use the same authorization mechanism as the standard Bonita
 **Note**: It is also possible to deploy a REST API extension using [REST portal API / "page" resource](portal-api.md#page).
 :::
 
+::: info
+**Note**: The deployement of the REST API extensions should always be made from the portal or its APIs. Notably, the createPage() method of the PageAPI will alllow you to deploy a REST API extension, but will not update the security settings, meaning that you won't be able to access it at runtime. 
```

Comment:
...from the portal or its APIs, ***not directly from Engine APIs***. Notably, the createPage() method of the ***Engine*** PageAPI ~will~ **would** alllow you to deploy a REST API extension, but ***would*** not update the security settings, meaning that you ***wouldn't*** be able to access it at runtime.